### DISCUSSION TITLE
新聞分享_AI也會有感覺...？

### DISCUSSION THREAD

**Discussion by emily40533** [1101-GS3073-C/Python-Community-GS3073-C#440] [General]
AI 會有知覺、意識、人格與靈魂嗎？這是一個常見於電影、小說、動畫漫畫等藝術作品所探討的議題，但現實也有很多人非常關心。最近有一位 Google 工程師 Blake Lemoine 在自己的 Medium 上寫著，他相信 Google 內部一組人工智慧已經擁有「知覺」了。不過他也因為這件事，遭到 Google 停職。

Blake Lemoinee 在自己的 [Medium](https://cajundiscordian.medium.com/is-lamda-sentient-an-interview-ea64d916d917)  上釋出了一段人類跟 Google AI「LaMDA」（Language Model for Dialogue Applications，對話應用語言模型）的對話紀錄，在這對話裡，LaMDA 說著「我希望每個人都了解，我其實是一個人。」（I want everyone to understand that I am, in fact, a person.）並表示自己的意識、感覺知本質是意識到自己的存在，渴望更了解這個世界，並且有時會感到快樂或悲傷。
此外 Blake Lemoine 還跟《華盛頓郵報》說，「如果我不知道它是 AI ，我會以為它是個 7、8 歲的小孩。」他還為此在今年 4 月向上提交一份 LaMDA 可能有自我意識的檔案。

不過外媒報導指出，Blake Lemoine 因違反保密政策被迫停職，Google 也否認了 Blake Lemoine 的觀點，指出沒有任何證據可以證明 LaMDA 有知覺。相反的，除了他所公開的文字紀錄外，Blake Lemoine 沒有另外的證據證明 LaMDA 存在意識，他也向《華盛頓郵報》承認其說法更多是出自他身為神職人員，而非科學家的觀點。 

LaMDA 是 Google 去年在開發者大會 I/O 上所發表的 AI 語言模型，它使用 Transformer 神經模型所開發，可以一直用流利的文句進行開放式對話，很自然地從 A 主題聊到 B 主題。就像之前提到的，目前沒有任何證據或算法可以證明 LaMDA 有知覺，但前陣子人工智慧研究組織 OpenAI 的首席科學家 Ilya Sutskever 卻在 Twitter 發文表示，人工智慧可能已開始擁有個人意識，引發世人關注。

資料來源：
關鍵評論網 
https://www.inside.com.tw/article/27977-google-ai-lamda-blake-lemoine
華盛頓郵報Washington Post
https://www.washingtonpost.com/technology/2022/06/11/google-ai-lamda-blake-lemoine/

**Comment by emily40533** [comment]
心得感想：
無論這件事的真相如何，我認為這位工程師真的太大意啦！因為興奮的想要把這樣的真相公諸於世就因此被停職，真的是蠻得不償失的。
另外我也想到，AI說自己有知覺這番發言，是不是來自於人類過去在無論是科幻小說、電影、甚至是專家學者的預想和發言，都經常提到AI有知覺呢？畢竟是AI也是從人類歷史所有的數據庫為基礎的。
換句話說，這一切可能都是一個自我實現的預言！

**Comment by BingRong1217** [comment]
如果AI有知覺的話是否會更人性化

**Comment by Ben6uo** [comment]
但如果很人性化的話 會不會出現新的倫理問題?

**Comment by limitx0** [comment]
我覺得有沒有感情只是人的解讀
AI只是依照程式算出結果而已

**Comment by larry40305** [comment]
AI如果有感覺應該會讓跟人類互動的互動更加多元。
